samples/matter/lock/README.rst

```diff
@@ -96,6 +96,29 @@ Instead, the factory reset and recommissioning to a Matter fabric allows the dev
 
 See `Matter door lock build types`_, `Selecting a build type`_, and :ref:`matter_lock_sample_switching_thread_wifi` for more information about how to configure and test this feature with this sample.
 
+.. matter_door_lock_sample_nrf70_firmware_patch_start
+
+A portion of the application code related to the nRF70 Wi-Fi firmware can be flashed into external memory to free up space in the on-chip memory.
+This option is available only when building for the nRF5340 DK with the nRF7002 EK shield attached.
+To prepare an application to use this feature, you need to create additional MCUBoot partitions.
+
+To learn how to configure MCUBoot partitions, see the :ref:`nrf70_fw_patch_update_adding_partitions` guide.
+
+To enable this feature, set the :kconfig:option:`CONFIG_NRF_WIFI_FW_PATCH_DFU` Kconfig option for both the Application and the MCUBoot child image.
```

Comment:
You no longer need that in the MCUBoot child image, right?

Reply:
Right, changed.